grc/gui_qt/components/window.py

```diff
@@ -448,6 +445,25 @@ def createActions(self, actions):
         actions["properties"].setEnabled(False)
 
         # View Actions
+        actions["zoom_in"] = Action(
+            Icons("zoom-in"),
+            _("Zoom &in"),
+            self,
+            shortcut="Ctrl+=",  # TODO: Keys.ZoomIn is "Ctrl++"
```

Comment:
I think we should use Keys.ZoomIn, to stay consistent with GRC Gtk and other applications